charts/aws-vpc-cni/templates/clusterrole.yaml

```diff
@@ -33,3 +33,7 @@ rules:
     resources:
       - '*'
     verbs: ["list", "watch"]
+  - apiGroups: ["", "events.k8s.io"]
+    resources:
+      - events
+    verbs: ["create", "patch", "list", "get"]
```

Comment:
why does it need `patch`?

Reply:
This is required for event aggregation. For eg, without `patch` we wouldn't see the aggregated info [x12 over 21s] below:
`Warning  MissingIAMPermissions  20s (x12 over 21s)  aws-node           Unauthorized operation ...`

FYI we removed `get` recently as it was not required.

Reply:
thanks for the explanation!